ods-tutorial/source/dashboard_modules/how_to_hide_or_show_widgets.rst

```diff
@@ -0,0 +1,529 @@
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    2 + Before going any further, declare the unique context of your page, using the `odsDatasetContext <https://help.opendatasoft.com/widgets/#/api/ods-widgets.directive:odsDatasetContext>`_ widget. This will allow our widgets to interact with one another -and it always is the proper way to build a dashboard.
+
+.. admonition:: Important
+   :class: important
+
+   Don't forget to delete the ``ods-dataset-context`` tag above the code of the chart widget. Also, replace the current context name (here, ``shanghaiworlduniversityrankingpublic``) by the new one (here, ``worlduniversityranking``).
+
+.. code-block:: html
+
+ <ods-dataset-context context="worlduniversityranking"
+                      worlduniversityranking-dataset="shanghai-world-university-ranking@public">
+
+ <div class="container">
+    <div class="ods-box">
+
+     <ods-chart align-month="true">
+        <ods-chart-query context="worlduniversityranking" field-x="country" maxpoints="20" sort="serie1-1">
+            <ods-chart-serie expression-y="total_score" chart-type="scatter" function-y="AVG" color="#FF515A" scientific-display="true">
+            </ods-chart-serie>
+        </ods-chart-query>
+     </ods-chart>
+
+    </div>
+ </div>
+
+ </ods-dataset-context>
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    3 + Add a filter related to the chosen dataset, using the `odsFacets <https://help.opendatasoft.com/widgets/#/api/ods-widgets.directive:odsFacets>`_ widget. Here, we retrieve the Country filter that already is an existing facet on the dataset published on Data.
+
+.. code-block:: html
+
+ <ods-dataset-context context="worlduniversityranking"
+                      worlduniversityranking-dataset="shanghai-world-university-ranking@public">
+
+ <div class="container">
+    <div class="ods-box">
+
+     <ods-chart align-month="true">
+        <ods-chart-query context="worlduniversityranking" field-x="country" maxpoints="20" sort="serie1-1">
+            <ods-chart-serie expression-y="total_score" chart-type="scatter" function-y="AVG" color="#FF515A" scientific-display="true">
+            </ods-chart-serie>
+        </ods-chart-query>
+     </ods-chart>
+
+    <ods-facets context="worlduniversityranking">
+      <ods-facet name="country"></ods-facet>
+    </ods-facets>
+
+    </div>
+ </div>
+
+ </ods-dataset-context>
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+.. rst-class:: block-step
+
+    4 + Click on :tutorial-keycap:`Preview`. Technically, everything works well: the chart indeed updates according to the chosen filter option. However, the filtered chart becomes completely useless, because it only displays one only point.
+
+.. rst-class:: img-hide
+
+    SCREENSHOT
+
+We need a second option visualization, that will only be displayed in case the filter is used. This second option visualization will replace the default, unfiltered chart that becomes useless once filtered. Let's do this!
```

Comment:
- `we need a second visualization option`? (twice here + more later), also, I don't really understand what it means. What about "alternative visualization" or "specific visualization"?
- `that will only be displayed if the filter is used`?